.github/workflows/rack-box-continuous.yml

```diff
@@ -19,125 +24,80 @@ jobs:
         path: RACK/rack-box/files/fuseki.tar.gz
         key: fuseki-3.16.0
 
-    - name: Cache RACK CLI
-      uses: actions/cache@v2
-      id: cache-rack-cli
-      with:
-        path: RACK/rack-box/files/rack-cli.tar.gz
-        key: rack-cli-${{ hashFiles('RACK/cli/**') }}
+    - name: Download Fuseki distribution
+      if: steps.cache-fuseki.outputs.cache-hit != 'true'
+      run: curl -LSs https://apache.claz.org/jena/binaries/apache-jena-fuseki-3.16.0.tar.gz -o RACK/rack-box/files/fuseki.tar.gz
+
+    - name: Download SemTK distribution
+      run: curl -LSs https://github.com/ge-semtk/semtk/releases/download/v2.3.0-20201103/semtk-opensource-v2.3.0-20201103-dist.tar.gz -o RACK/rack-box/files/semtk.tar.gz
+
+    - name: Download style spreadsheet
+      run: curl -LSs https://github.com/KrauseFx/markdown-to-html-github-style/raw/master/style.css -o RACK/rack-box/files/style.css
+
+    - name: Download systemctl script
+      run: curl -LSs https://github.com/gdraheim/docker-systemctl-replacement/raw/v1.5.4260/files/docker/systemctl3.py -o RACK/rack-box/files/systemctl3.py
 
     - name: Cache RACK CLI dependencies
-      if: steps.cache-rack-cli.outputs.cache-hit != 'true'
       uses: actions/cache@v2
       with:
         # This path is specific to Ubuntu
         path: ~/.cache/pip
+        # Look to see if there is a cache hit for the corresponding requirements file
         key: ${{ runner.os }}-pip-${{ hashFiles('RACK/cli/requirements.txt') }}
         restore-keys: |
           ${{ runner.os }}-pip-
           ${{ runner.os }}-
 
-    - name: Cache RACK ontology and data
-      uses: actions/cache@v2
-      id: cache-rack-ontology
-      with:
-        path: RACK/rack-box/files/rack.tar.gz
-        key: rack-ontology-${{ hashFiles('RACK/RACK-Ontology/**', 'RACK/nodegroups/**') }}
-
-    - name: Cache RACK documentation
-      uses: actions/cache@v2
-      id: cache-rack-doc
-      with:
-        path: |
-          RACK/rack-box/files/documentation.html
-          RACK/rack-box/files/index.html
-        key: rack-doc-${{ hashFiles('RACK.wiki/*.md') }}
-
-    - name: Cache SemTK distribution
-      uses: actions/cache@v2
-      id: cache-semtk
+    - name: Set up Python
+      uses: actions/setup-python@v2
       with:
-        path: RACK/rack-box/files/semtk.tar.gz
-        key: semtk-2.2.2-SNAPSHOT-v3
-
-    - name: Cache style spreadsheet
-      uses: actions/cache@v2
-      id: cache-style
-      with:
-        path: RACK/rack-box/files/style.css
-        key: style-db9e23f
-
-    - name: Cache systemctl script
-      uses: actions/cache@v2
-      id: cache-systemctl
-      with:
-        path: RACK/rack-box/files/systemctl3.py
-        key: systemctl-1.5.4260
-
-    - name: Download Fuseki distribution
-      if: steps.cache-fuseki.outputs.cache-hit != 'true'
-      run: curl -LSs https://apache.claz.org/jena/binaries/apache-jena-fuseki-3.16.0.tar.gz -o RACK/rack-box/files/fuseki.tar.gz
+        python-version: 3.8
```

Comment:
We have to specify python-version 3.8; we can't install a '3.x' (actually 3.9.0) wheel in our rack-box image because Ubuntu 20.04's native python is 3.8, not 3.9.